### PR TITLE
Intake Form Step 3: Confirmation - Linked Appeals & New Tasks

### PR DESCRIPTION
**Pull request by Rnmarshall93**
Resolves [Intake Form Step 3: Confirmation - Linked Appeals & New Tasks](https://jira.devops.va.gov/browse/APPEALS-29428)

# Description
Added table summary in step 3 to display information about tasks related to appeals.

## Acceptance Criteria
- [X] Code compiles correctly

## Testing Plan
1. Navigate to Correspondence and select a batman veteran.
2. Add tasks related to an appeal, make sure you remember whether or not you select the 'evidence waived' option.
3. navigate to the confirmation page
4. You will see the tasks related to appeal summary at the bottom of the page. It will display the following

- A badge for the appeal type
- A hyperlink that will take you to the appeal
- Information about the task (currently active task, evidence window waived, assigned to)
- Edit Section hyperlink that will return the user to step 2
- Information about all additional tasks.

- [ ] # Frontend

## User Facing Changes
 - [x] Screenshots of UI changes added to PR & Original Issue

 BEFORE|AFTER
 
image
|
image

### CODE REVIEW

File: client/app/queue/correspondence/intake/components/ConfirmCorrespondence/ConfirmTasksRelatedToAnAppeal.jsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        <tr>
+          <td style={borderlessTd}>
+            <div style={{ width: 'fit-content',
+              padding: '3px',
+              backgroundColor: 'white',
+              border: `1px solid ${COLORS.COLOR_COOL_BLUE_LIGHTER}`,
+              whiteSpace: 'nowrap'
+            }}>
+              <DocketTypeBadge name={(fetchedAppeals.find((appeal) => appeal.id === task).docketName)} />
+              <LinkToAppeal appealExternalId={fetchedAppeals.find((appeal) => appeal.id === task).externalId}>
```
Comment (HunJerBAH) [inline client/app/queue/correspondence/intake/components/ConfirmCorrespondence/ConfirmTasksRelatedToAnAppeal.jsx:88]: Small change, but the link to appeal needs to open in a new window.

**Review by HunJerBAH — COMMENTED** [review]
small fix but otherwise fantastic work on this!

**Review by jimfoley25 — APPROVED** [review]
sweet

**Review by HunJerBAH — APPROVED** [review]
Thank you for the changes! Fantastic job!!

**Pull request merged**